packages/styles/src/token.scss

```diff
@@ -3,58 +3,64 @@
 
 $block: #{$fd-namespace}-token;
 
+@mixin fd-readonly {
+  --fdToken_Text_Color: var(--sapContent_LabelColor);
+  --fdToken_Padding_Inline: var(--fdToken_ReadOnly_Padding_Inline);
+  --fdToken_Border: var(--sapButton_BorderWidth) solid var(--fdToken_Border_Color_Read_Only);
+
+  background-color: var(--fdToken_Background_Color_Read_Only);
+
+  @include fd-compact-or-condensed() {
+    --fdToken_Padding_Inline: 0.25rem;
+  }
+
+  @include fd-hover() {
+    --fdToken_Text_Color: var(--sapContent_LabelColor);
+    --fdToken_Border: var(--sapButton_BorderWidth) solid var(--fdToken_Border_Color_Read_Only);
+
+    background-color: var(--fdToken_Background_Color_Read_Only);
+  }
+}
```

Comment:
I'd call this fd-readonly-token or something along the lines of specifying the context of the token. fd-readonly sounds like a something that should not belong to the token.scss file

Reply:
done